output/components.eslint.txt

```diff
@@ -4,6 +4,9 @@
 
 The react/require-extension rule is deprecated. Please use the import/extensions rule from eslint-plugin-import instead.
 
+/home/travis/build/Talend/ui/packages/components/src/ActionBar/ActionBar.component.js
+  5:27  error  Absolute imports should come before relative imports  import/imports-first
```

Comment:
plz, remove this error

Reply:
it's fix in the commit https://github.com/Talend/ui/pull/1237/commits/54c2c068537206fab2e611f72484c29ec446aea5